examples/_example_aws_lambda_handler.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) MongoDB, Inc. 2022-present.
```

Comment:
Is the `_` prefix of the filename intended? Can this be `example_aws_lambda_handler.go`?

If it is intended, I suggest leaving a comment to describe why. Is it to avoid adding the `lambda` dependency?

Reply:
I prefixed `_` in the filenames under the `examples` folder, including "_example_customdns_test.go", to prevent the examples from being compiled. These examples contain third-party packages, such as `lambda`, which we do not want to include in the driver.